src/main/java/org/apache/sling/junit/impl/servlet/JacocoServlet.java

```diff
@@ -83,8 +111,10 @@
     @Reference
     private HttpService httpService;
 
-    protected void activate(ComponentContext ctx) throws ServletException, NamespaceException {
-        servletPath = getServletPath(ctx);
+    @Activate
+    @Modified
```

Comment:
remove

Reply:
Fixed in the latest changes. Will do another pull request.